Stepper: describe API (#7166)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxStepper/1 Configuration/linear.md

---
---
##### shortDescription
Specifies a navigation mode (linear/non-linear). 

---

> If this property is `true`, navigation becomes sequential, allowing users to proceed step-by-step. If `false`, navigation is non-linear, permitting users to skip steps.

Reviewer: "allowing to proceed step-by-step": the verb "allow" doesn't really fit in here. It makes it seem like the component gives users some freedom, when it fact it restricts them to a certain way of navigation. 

I would actually just focus on the "non-linear" part. If we say it allows to skip steps, then it's obvious that the other value doesn't allow to skip steps. 

```suggestion
If this property `false`, navigation is non-linear. In other words, users can skip steps. 
```

> [note] You cannot skip a [disabled](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/#disabled) step in the linear mode.

Reviewer 2: ```suggestion
[note] You cannot skip a [disabled](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/#disabled) step in linear mode.
```
